### PR TITLE
Switch Daytrotter connector to v2 format

### PR DESCRIPTION
**Pull request by alexesprit**

### CODE REVIEW

File: core/connectors.js [hunk 1]
```diff
@@ -230,8 +230,9 @@ define(function() {
 
 		{
 			label: 'Daytrotter',
-			matches: ['*://www.daytrotter.com/*'],
-			js: ['connectors/daytrotter.js']
+			matches: ['*://www.daytrotter.com*'],
```
Comment (inverse) [inline core/connectors.js]: Should it have the trailing / before the * or does it not matter?

File: connectors/v2/daytrotter.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+/**
+ * Example: http://www.daytrotter.com/radio.html?playlistid=1239601&trackid=4893828
+ */
+function isRadioPlayer() {
+	return $('body').hasClass('radio');
+}
+
+function setupConnector() {
```
Comment (inverse) [inline connectors/v2/daytrotter.js]: Super clear :+1:

**Comment by inverse** [comment]
Aside the comments looks good :+1:

File: core/connectors.js (hunk 1 above)
Comment (alexesprit) [inline core/connectors.js]: It should be fixed, of course.

**Comment by alexesprit** [comment]
Hmm, Daytrotter Radio and Paste Magazine Radio have the same layout.

**Comment by inverse** [comment]
Interesting. But this one implements more functionality right?

**Comment by alexesprit** [comment]
Yes, this connector covers Daytrotter Radio (copy of PasteMagazine Radio) and Daytrotter itself.

Of course, we can drop `Add Daytrotter Radio support` commit and use `pasteradio` connector for Daytrotter Radio, but it's such a mess.

So, the PR is ready for merge.

**Comment by inverse** [comment]
:+1:

**Pull request merged**